toolchest_client/tools/kraken2.py

```diff
@@ -32,7 +32,26 @@ def __init__(self, tool_args, output_name, inputs, output_path,
             output_is_directory=True,
         )
 
-    def _sanity_output_check(self):
+    def _preflight(self):
```

Comment:
At some point, can you add a comment line about what specifically the Kraken 2 preflight is doing? 

(It's evident right now, but this will help for looking back at this code later.)

Reply:
We can! I've added this blurb to the ongoing `fix/update-docs` branch:

```
    def _preflight(self):
        """
        Kraken 2 specific preflight.
        
        Checks:
        - that `output_path` either: a) exists and isn't a file, or b) can and does create the output path.
        - if the files that Kraken 2 will write already exist. If they do, prints a warning.
        """
        super()._preflight()
```

That said, I'm actually in favor of not adding a comment. I could grab the above context from the above code more easily than by reading a comment, although I'm likely biased.